dev/io.openliberty.microprofile.openapi.4.1.internal_fat_tck/build.gradle

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2024, 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+apply from: '../wlp-gradle/subprojects/maven-central-mirror.gradle'
+
+dependencies {
+  requiredLibs 'org.hamcrest:hamcrest:2.1'
+}
+
+addRequiredLibraries.dependsOn addJakartaTransformer
```

Comment:
Can remove this when #31324 is done

Reply:
Do we use fattest.simplicity to install applications?  If no, is it really needed?

Reply:
Good point, it shouldn't be.